test/integration/default/files/_mapdata/centos-8.yaml

```diff
@@ -0,0 +1,29 @@
+# yamllint disable rule:indentation rule:line-length
+# CentOS Linux-8
```

Comment:
Technically, `CentOS Linux-8` is EOL and has been removed from the CI, so this should be:

```suggestion
# CentOS Stream-8
```

However, since this is only a comment and has no bearing on the actual tests, we can ignore this.

Reply:
@myii Here is the fix https://github.com/saltstack-formulas/consul-formula/pull/57

Reply:
> @myii Here is the fix #57

Thanks, merged it.